src/components/sustainability/HeroSection.tsx

```diff
@@ -30,21 +30,83 @@ const FeatureSection = ({ textContent }: HeroSectionsProps) => {
         </div>
 
         {/* Display goals */}
-        <div className="relative grid grid-cols-2 justify-items-center pt-6">
-          {goals.map((goal, index) => {
-            const bgColor = index === 1 ? 'bg-green-25' : index === 4 ? 'bg-green-55' : 'bg-green-40';
+        <div className="grid grid-cols-3 pt-6 hidden lg:grid">
+          {goals.slice(0, 3).map((goal, index) => {
```

Comment:
Instead of slicing multiple times and duplicating all this code, it might be better to directly add the unique parameters of each goal to the array (similar to how the icon is handled). This way, you can reuse the same code and prevent it from becoming increasingly complex over time.
What do you think?

Reply:
Yes, its a better idea, i will try

Reply:
Done